src/app/components/common/ImageDiv/ImageDiv.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
 import Grid from '@mui/material/Grid';
 import { observer } from 'mobx-react';
 import { useStyles } from './imageDiv.styles';
```

Comment:
Full path. Also style file should start with Uppercase

Reply:
done